source/developers-guide/rest-api/examples/media/index.md

```diff
@@ -0,0 +1,207 @@
+            )
+        )
+    ),
+    'configuratorSet' => array(
+        'name' => 'Test-Set',
+        'groups' => array(
+            array(
+                'id' => 5,
+                'name' => 'Flascheninhalt',
+                'options' => array(
+                    array('id' => 11, 'name' => '0,2 Liter'),
+                    array('id' => 35, 'name' => '0,5 Liter'),
+                )
+            )
+        )
+    ),
+    'variants' => array(
+        array(
+            'number' => 'swTEST52b04c97dd280',
+            'inStock' => 100,
+            'unitId' => 1,
+            'prices' => array(
+                array('customerGroupKey' => 'EK','from' => 1,'to' => '-','price' => 400)
+            ),
+            'configuratorOptions' => array(
+                array('option' => '0,2 Liter', 'groupId' => 5)
+            )
+        ),
+        array(
+            'number' => 'swTEST52b04c97dd28f',
+            'inStock' => 100,
+            'unitId' => 1,
+            'prices' => array(
+                array('customerGroupKey' => 'EK', 'from' => 1, 'to' => '-', 'price' => 400)
+            ),
+            'configuratorOptions' => array(
+                array('option' => '0,5 Liter', 'groupId' => 5)
+            )
+        )
+    )
+);
+```
+
+### The following example creates two assignments for the passed image.
+* Show the image if the customer selects <code>"0,5 Liter"</code> and "Rot".
+* Show the image if the customer selects "Blau".
+
+```php
+array(
+    'name' => 'Testartikel',
+    'images' => array(
+        array(
+            'mediaId' => 236,
+            'options' => array(
+                array(
+                    array('name' => '0,5 Liter'),
+                    array('name' => 'rot')
+                ),
+                array(
+                    array('name' => 'blau')
+                )
+            ),
+        ),
+    ),
+    'configuratorSet' => array(
+        'name' => 'Test-Set',
+        'groups' => array(
+            array(
+                'id' => 5,
+                'name' => 'Flascheninhalt',
+                'options' => array(
+                    array('id' => 11,'name' => '0,2 Liter'),
+                    array('id' => 35,'name' => '0,5 Liter'),
+                    array('id' => 12,'name' => '0,7 Liter'),
+                    array('id' => 32,'name' => '1,0 Liter'),
+                ),
+            ),
+            array(
+                'id' => 6,
+                'name' => 'Farbe',
+                'options' => array(
+                    array('id' => 13,'name' => 'weiss'),
+                    array('id' => 14,'name' => 'schwarz'),
+                    array('id' => 15,'name' => 'blau'),
+                    array('id' => 28,'name' => 'rot'),
+                ),
+            ),
+        ),
+    ),
+    'variants' => array(...)
+);
+```
+
+### Assignment
+ 
+The first level of the option array defines how many assignment are created. You create within each assignment how to assign. 
```

Comment:
what do you want to say with this: ???
You create within each assignment how to assign